vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/VssDefinitionProcessor.kt

```diff
@@ -100,11 +100,11 @@ class VssDefinitionProcessor(
         private fun loadAssetFile(fileName: String): File? {
             val generatedFile = codeGenerator.generatedFile.firstOrNull() ?: return null
             val generationPath = generatedFile.absolutePath
-            val buildPath = generationPath.replaceAfterLast(buildDir, "")
-            val assetsFilePath = buildPath + fileSeparator + assetsDir
-            val assetsFolder = File(assetsFilePath)
+            val buildPath = generationPath.replaceAfter("$BUILD_FOLDER_NAME$fileSeparator", "")
```

Comment:
Can you elaborate on the changes here? It feels like you are reopening this issue here: #67 

This breaks F-Droid and potentially all CIs who put their project somewhere in a build folder: someDir/build/org.eclipse.kuksa.sdk/

Reply:
Fixed, lost during rebase. :)